### PR TITLE
fixes #13189 - allows authentication with username and certificate

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Comment by ehelms** [comment]
MInd changing the commit message a bit to not include capsules since they are not part of Pulp itself? That way it better captures what this enables with respect to Pulp access.

**Comment by cfouant** [comment]
@ehelms - updated commit message, thanks

File: templates/etc/httpd/conf.d/_ssl_username.conf.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#
+# Apache configuration file for Pulp's Puppet support
```
Comment (ehelms) [inline templates/etc/httpd/conf.d/_ssl_username.conf.erb]: This seems incorrect documentation, I think you'd  be OK reducing this to a one line comment above the directive.

**Comment by ehelms** [comment]
ACK - thanks @cfouant !

**Pull request merged**